engine/src/fungible_token.rs

```diff
@@ -11,8 +11,10 @@ use crate::prelude::{
 };
 use aurora_engine_sdk::io::{StorageIntermediate, IO};
 
+/// Gas for `resolve_transfer`: 5 TGas
 const GAS_FOR_RESOLVE_TRANSFER: NearGas = NearGas::new(5_000_000_000_000);
-const GAS_FOR_FT_ON_TRANSFER: NearGas = NearGas::new(10_000_000_000_000);
+/// Gas for `ft_on_transfer`
+const GAS_FOR_FT_TRANSFER_CALL: NearGas = NearGas::new(25_000_000_000_000);
```

Comment:
Question: how did you determine this value?

Reply:
Experimentally by tests.

Reply:
But before that it was:  `GAS_FOR_FINISH_DEPOSIT - GAS_FOR_FT_ON_TRANSFER = 50 TGas - 10 TGas = 40 TGas`.
Currently: `GAS_FOR_FT_TRANSFER_CALL + GAS_FOR_FT_ON_TRANSFER = 30 TGas`
So `GAS_FOR_FT_ON_TRANSFER` was increased anyway.

Reply:
Yes, I agree it is clear that `ft_on_transfer` is getting much more gas now, and that is good.

What I am asking about is if our own `ft_transfer_call` will ever use more than 25 Tgas. If it does then creating the promise will fail because we won't have enough prepaid gas to cover it all.

Reply:
I thinking about that. To guarantee the same work as it does now, this constant must be: `35 TGas`